src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-troubleshooting.mdx

```diff
@@ -52,6 +52,16 @@ You can also check for integration errors by querying [NrIntegrationError events
 
 Various parts of the New Relic UI rely on the presence of specific attributes to function properly. You can use the [NRQL console](/docs/query-your-data/explore-query-data/query-builder/nrql-console/) feature in many places to check the `WHERE` or `FACET` clauses of the query for required attributes. You can also edit those clauses and re-run the query to determine whether there is data present with those attributes missing. Examples of required attributes include `service.name` and `service.instance.id`. For a more complete list of examples, see [resources](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts/#resources).
 
+## OpenTelemetry log correlation isn't working
+
+### Problem
+
+You've correlated your logs with your service using `service.id`([OpenTelemetry logs: Best practices](content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs)) so you can see logs associated with traces, but you don't see any logs in the New Relic UI. In this scenario, the log data has made it to New Relic, but isn't showing up in the distributed trace UI with corresponding spans.
+
+### Solution
+
+To correlate with trace data, the logs need to include the trace context that is contained `trace_id` and `span_id`. However, to ensure your logs show up in the New Relic UI, you'll need to configure rules in your log pipeline to translate `trace_id` to `trace.id` and `span_id` to `span.id`.
```

Comment:
```suggestion
To correlate with trace data, the logs need to include the trace context that is contained in `trace_id` and `span_id`. However, to ensure your logs show up in the New Relic UI, you'll need to configure rules in your log pipeline to translate `trace_id` to `trace.id` and `span_id` to `span.id`.
```

Is that the right preposition here?

Reply:
I prefer slightly something like: "translate `trace_id` and `span_id` to `trace.id` and `span.id`"